src/listener.h

```diff
@@ -37,6 +37,9 @@ struct Listener {
     /* Configuration fields */
     struct Address *address, *fallback_address, *source_address;
     const struct Protocol *protocol;
+#ifdef SO_REUSEPORT
```

Comment:
If we treat `resuseport yes` as a valid configuration option on unsupported platforms this and `accept_listener_reuseport` will not need ifdef guards.